index.js

```diff
@@ -0,0 +1,12 @@
+function sumar(a,b,c) {
```

Comment:
se le puede dar un nombre mas descriptivo como por ejemplo sumarTresNumeros, Verificar que pasa si sele pasa uno o dos parametros en lugar de tres. Contemplar si es necesario implementar valores por default.

Reply:
Se modificó en nombre de la funcion por "sumarTresNumeros".
Si se le pasa 2 parametros el test da error:
> practica_1@1.0.0 test
> jest

 FAIL  ./index.spec.js
  ✕ cuando sumo los tres numeros entonces el resultado es: 4 (5 ms)

  ● cuando sumo los tres numeros entonces el resultado es: 4

    expect(received).toBe(expected) // Object.is equality

    Expected: 4
    Received: NaN

      2 |
      3 | test('cuando sumo los tres numeros entonces el resultado es: 4', () => {
    > 4 |     expect(sumarTresNumeros(1, 3)).toBe(4);
        |                                    ^
      5 |   });

      at Object.<anonymous> (index.spec.js:4:36)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        0.729 s, estimated 1 s
Ran all test suites.

Se  le agregó valor por default y el test funcionó correctamente
function sumarTresNumeros(a,b,c) {
    
    c = (typeof c !== 'undefined') ?  c : 4
    const d = a + b + c;

    return d;
    
}

module.exports = { sumarTresNumeros };